lib/bambora/adapters/json_response.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+module Bambora
+  ##
+  # Parses a JSON response into a Hash
+  class JSONResponse < Response; end
```

Comment:
Mostly because I had written this class before I moved all this into a directory. I considered adding a namespace, but I was focused on getting this feature done. Happy to add this as a tidy up later.